pubsub/solace/amqp/metadata.yaml

```
@@ -0,0 +1,64 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: Solace-AMQP
+version: v1
+status: beta
+title: "solace-amqp"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-solace-amqp/
+capabilities:
+  - ttl
+authenticationProfiles:
+  - title: "Solace authentication"
+    description: "Authenticate using Solace credentials"
+    metadata:
+      - name: username
+        required: true
+        sensitive: false
+        description: "The username to connect to the broker"
+        example: '"default"'
+        type: string
+      - name: password
+        required: true
+        sensitive: true
+        description: "The password to connect to the broker"
+        example: '"default"'  
+        type: string
+  - title: "No authentication"
+    description: "Connect to the broker without credential validation"
+    metadata:
+      - name: anonymous
+        required: false
+        description: |
+          To connect to the broker without credential validation. Only works if enabled on the broker. A username and password would not be required if this is set to true.
+        example: '"true"'
+        type: bool
+metadata:
+  - name: url
+    required: true
+    description: |
+      Address of the AMQP broker. Can be secretKeyRef to use a secret reference.
+    example: '"amqp://host.domain[:port]"'
+    type: string   
+  - name: caCert
+    required: true
+    description: |
+      Certificate Authority (CA) certificate in PEM format for verifying server TLS certificates.
+    example: "-----BEGIN CERTIFICATE-----\n<base64-encoded DER>\n-----END CERTIFICATE-----"
+    type: string
+  - name: clientCert
+    required: true
+    sensitive: true
```

Not required - using TLS is optional (also not sensitive)

```suggestion
    required: false
```